### DISCUSSION TITLE
[Windows] Add Kreya to Winget

### DISCUSSION THREAD

**Discussion by Panda-Sharp** [riok/Kreya#177] [Ideas]
Would be great to be able to install and update Kreya via Winget ( Windows Package Manager https://github.com/microsoft/winget-cli )

> E.G.  winget install kreya

Thanks!

**Comment by ni507** [comment]
We will probably not support winget in the near future, as it is a little tedious to support and maintain a large number of package manager integrations. See also https://github.com/riok/Kreya/issues/150

However, feel free to add Kreya to winget yourself or make a request there. All informations about the latest version is available here: https://stable-downloads.kreya.app/appcast.json

- Windows x64 https://stable-downloads.kreya.app/Kreya-win-x64.msi
- Windows arm64 https://stable-downloads.kreya.app/Kreya-win-arm64.msi

**Comment by CommonGuy** [comment]
Kreya has been added to Winget: https://github.com/microsoft/winget-pkgs/tree/master/manifests/r/riok/Kreya/1.17.0
